packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -4,10 +4,16 @@
       <div v-if="link.path">
         <router-link
           :to="link.path"
-          exact
+          v-slot="{ route, href, navigate }"
           class="tree-nav-link"
+        >
+          <a
+            :href="href"
+            @click="navigate"
+            :class="route.path === $route.path ? 'router-link-active' : ''"
+            :aria-current="route.path === $route.path && 'page'"
```

Comment:
would appreciate a hint on why `&& 'page'` condition is required

Reply:
aria is a set of attr that using mostly for screen readers, to make resources more accessible especially for people with disabilities,   aria-current="page" used for indication of the current active page, so if  `route.path === $route.path` `true` then this attr will populated with `page` value. This code was copied from github thread where was discussed issue of `exect` attr in the case when hash routings are using. In our case, since we not supporting such a feature, this code can be removed. Or maybe any other thoughts?

Reply:
thanks - I am fine with keeping it since it follows the original RouterLink implementation